src/index.html

```diff
@@ -10,6 +10,9 @@
   <meta name="description" content="Portfolio for Chase Ottofy">
   <meta name="keywords" content="chase ottofy portfolio, portfolio, chase, ottofy, chaseottofy, chase ottofy, chase ottofy portfolio, front end developer, front end, javascript, react developer, react">
   <title>Ottofy, Chase</title>
+  <link href="./favicon.svg" rel="icon">
+  <link href="./styles/style.scss" rel="stylesheet">
+  <script src="./index.js" defer="defer"></script>
```

Comment:
You can specify source files of images, scripts, styles directly in HTML

Reply:
Thank you this is excellent!